JavaScript/script.js

```diff
@@ -0,0 +1,62 @@
+const featuredCards = document.querySelector('.featured-speaker');
+
+const cards = [
+  {
+    title: 'Hamid Karzai',
+    image: '../images/Karzai.png',
+    shortDescription: 'The fourth president of Afghanistan',
+    longDescription:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  },
+  {
+    title: 'Abdullah Abdullah',
+    image: '../images/Abdulah.png',
+    shortDescription: 'Lorem ipsum dolor, sit amet.',
+    longDescription:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  },
+  {
+    title: 'Mohammad Mohaqiq',
```

Comment:
- Kindly Fix the images under this section.
- 
![Capture](https://user-images.githubusercontent.com/63972670/205440628-9cd42821-b6f6-4c39-a310-44106288cb64.PNG)